go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -269,6 +269,23 @@ var metricEgressFirewallCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The number of egress firewall policies",
 })
 
+/** AdminNetworkPolicyMetrics Begin**/
+var metricANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
```

Comment:
can you update the metrics doc we have to include the new metrics here in ovn-k?

Reply:
oh yes forgot about that sorry, doing this now..

Reply:
done! added this PR to the changelog